JuiceMaker/JuiceMaker/Controller/JuiceMakerViewController.swift

```diff
@@ -0,0 +1,120 @@
+        presentStockManager()
+    }
+    
+    @IBAction func touchUpJuiceOrderButton(_ sender: UIButton) {
+        guard let juice = juiceButtonsDictionary[sender] else { return }
+        
+        do {
+            try juiceMaker.makeJuice(for: juice)
+        } catch {
+            presentOrderFailureAlert()
+            return
+        }
+        presentOrderSuccessAlert(juice: juice)
+        updateFruitCountLabels(juice: juice)
+    }
+    
+    func defineDictionary() {
+        juiceButtonsDictionary = [
+            strawberryJuiceButton: .strawberryJuice,
+            bananaJuiceButton: .bananaJuice,
+            kiwiJuiceButton: .kiwiJuice,
+            pineappleJuiceButton: .pineappleJuice,
+            mangoKiwiJuiceButton: .mangoKiwiJuice,
+            mangoJuiceButton: .mangoJuice,
+            strawberryBananaJuiceButton: .strawberryBananaJuice
+        ]
+        
+        fruitLabelsDictionary = [
+            .strawberry: strawberryCountLabel,
+            .banana: bananaCountLabel,
+            .kiwi: kiwiCountLabel,
+            .pineapple: pineappleCountLabel,
+            .mango: mangoCountLabel
+        ]
+    }
+    
+    func configureFruitCountLabels() {
+        for fruit in Fruit.allCases {
+            fruitLabelsDictionary[fruit]?.text = fruitStore.getStockCountToString(of: fruit)
+        }
+    }
+    
+    func updateFruitCountLabels(juice: Juice) {
+        for fruit in juice.recipe.keys {
+            fruitLabelsDictionary[fruit]?.text = fruitStore.getStockCountToString(of: fruit)
+        }
+    }
+    
+    func presentStockManager() {
+        guard let stockManagerNC = self.storyboard?.instantiateViewController(withIdentifier: "StockManagerNavigationController") as? UINavigationController else { return }
+                        
+        self.present(stockManagerNC, animated: true)
+    }
+    
+    func presentOrderSuccessAlert(juice: Juice) {
+        let message = "\(juice.rawValue) 나왔습니다! 맛있게 드세요!"
+               
+        let alert = UIAlertController(title: "알림",
+                                      message: message,
+                                      preferredStyle: .alert)
+        let okAction = UIAlertAction(title: "OK", style: .default)
+        
+        alert.addAction(okAction)
+        
+        present(alert, animated: true, completion: nil)
+    }
+    
+    func presentOrderFailureAlert() {
+        let alert = UIAlertController(title: "알림",
+                                      message: "재료가 모자라요. 재고를 수정할까요?",
+                                      preferredStyle: .alert)
+        let okAction = UIAlertAction(title: "예", style: .default) { _ in
+            self.presentStockManager()
+        }
+        let cancelAction = UIAlertAction(title: "아니오", style: .destructive)
+        
+        alert.addAction(okAction)
+        alert.addAction(cancelAction)
+        
+        present(alert, animated: true, completion: nil)
```

Comment:
참고로 `completion`으로 처리할 코드가 없다면 아래처럼 생략 가능합니다~
```suggestion
        present(alert, animated: true)
```

Reply:
넵 사용하지 않는 컴플리션은 제거했습니다!